### DISCUSSION TITLE
Renovate doesn't update bazel deps

### DISCUSSION THREAD

**Discussion by neee** [renovatebot/renovate#14448] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

We used Renovate for the Gradle project, then did a migration from Gradle to Bazel the same project and after this step, I haven't seen Renovate PRs with updates.

I created a sample BAZEL project inside my personal repository:
https://github.com/neee/java-bazel-sample

Added Renovate base config:
`{
    "extends": [
        "config:base"
    ],
    "schedule": [
        "before 5am every weekday"
    ]
}`

For this sample project, I also haven't seen any updates.

Maybe I missed something?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment]
Isn't this related to https://github.com/renovatebot/renovate/issues/5151 ?
